woocommerce/wc-admin
# Hiding primary tabs on Settings within WooCommerce since it's redunda…

Fixes #5572

The given issue surfaced the fact that the top (primary) tabs within Settings were actually redundant with the new navigation, and it was decided that the tabs should be hidden when navigation is active. 

I also discovered that there were a couple pages missing from the navigation that would have been inaccessible after the change, so I added those in as well. You'll find them as "Logs" and "Scheduled Actions" under "Tools."

There is a small style tweak to the secondary tabs (which appear in "Settings -> Advanced") to make them a bit more prominent with some additional whitespace. These will see further visual changes in a future PR, as discussed in the issue. 

### Screenshots

**Settings -> Advanced without the primary tabs:**

![image](https://user-images.githubusercontent.com/444632/99115363-2080a500-25a7-11eb-92e7-a98a3c271462.png)

**Tools menu with added items:**

![image](https://user-images.githubusercontent.com/444632/99115468-4ad26280-25a7-11eb-960d-b6c1cacfee74.png)


### Detailed test instructions:

- Checkout branch
- Ensure new navigation feature is enabled
- Navigate to WooCommerce -> Settings -> Advanced
- Observe the the top tabs are not visible, and that the secondary tabs still are
- Navigate to WooCommerce -> Tools
- Observe that we now have items for **Logs** & **Scheduled Actions**
